contrib/azurerm/group_vars/all

```diff
@@ -17,12 +18,29 @@ minions_os_disk_size: 1000
 
 admin_username: devops
 admin_password: changeme
+
+# MAKE SURE TO CHANGE THIS TO YOUR PUBLIC KEY to access your azure machines
 ssh_public_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDLRzcxbsFDdEibiyXCSdIFh7bKbXso1NqlKjEyPTptf3aBXHEhVil0lJRjGpTlpfTy7PHvXFbXIOCdv9tOmeH1uxWDDeZawgPFV6VSZ1QneCL+8bxzhjiCn8133wBSPZkN8rbFKd9eEUUBfx8ipCblYblF9FcidylwtMt5TeEmXk8yRVkPiCuEYuDplhc2H0f4PsK3pFb5aDVdaDT3VeIypnOQZZoUxHWqm6ThyHrzLJd3SrZf+RROFWW1uInIDf/SZlXojczUYoffxgT1lERfOJCHJXsqbZWugbxQBwqsVsX59+KPxFFo6nV88h3UQr63wbFx52/MXkX4WrCkAHzN ablock-vwfs@dell-lappy"
 
-kube_apiserver_port: 6443
+# Disable using ssh using password. Change it to false to allow to connect ot ssh by password
+disablePasswordAuthentication: true
 
 # Azure CIDRs
 azure_vnet_cidr: 10.0.0.0/8
 azure_admin_cidr: 10.241.2.0/24
 azure_masters_cidr: 10.0.4.0/24
 azure_minions_cidr: 10.240.0.0/16
+
+# Azure loadbalancer port to use to access your cluster
+kube_apiserver_port: 6443
+
+# Azure Netwoking and storage naming to use with inventory/all.yml
+virtual_network_name: KubeVNET
+subnet_admin_name: ad-subnet
```

Comment:
Just a small proposal, maybe it's better to declare this variables like:
`subnet_admin_name: "{{ azure_subnet_admin_name|d('ad-subnet') }}"`
and so on?

This will give an ability to have in ansible inventory something like:

```
[kubernetes:vars]
azure_subnet_admin_name="any-ad-subnet" 
```
for declaration via inventory file without actual changing in version controlled files?